### PR TITLE
Task/upgrade xena

### PR DESCRIPTION
**Pull request by AlvaroVega**
This version based on xena is needd to upgrade keystone spassword 1.17 (based on stein) to keystone based on antelope

### CODE REVIEW

File: docker/postlaunchconfig.sh [hunk 1]
```diff
@@ -116,8 +116,8 @@ if [ "${LOG_LEVEL}" == "DEBUG" ]; then
     DEFAULT debug True
     openstack-config --set /etc/keystone/keystone.conf \
     DEFAULT insecure_debug True
-    openstack-config --set /etc/keystone/keystone.conf \
-    wsgi debug_middleware True
+    # openstack-config --set /etc/keystone/keystone.conf \
+    # wsgi debug_middleware True
```
Comment (fgalan) [inline docker/postlaunchconfig.sh:120]: Leftover comments?

File: docker/postlaunchconfig_update.sh [hunk 2]
```diff
@@ -89,8 +89,8 @@ if [ "${LOG_LEVEL}" == "DEBUG" ]; then
     DEFAULT debug True
     openstack-config --set /etc/keystone/keystone.conf \
     DEFAULT insecure_debug True
-    openstack-config --set /etc/keystone/keystone.conf \
-    wsgi debug_middleware True
+    # openstack-config --set /etc/keystone/keystone.conf \
+    # wsgi debug_middleware True
```
Comment (fgalan) [inline docker/postlaunchconfig_update.sh:93]: Leftover comments?

File: docker/cracklib-devel-2.9.6-15.el8.x86_64.rpm
Comment (fgalan) [inline docker/cracklib-devel-2.9.6-15.el8.x86_64.rpm:1]: This file has been added intentionally? (we don't use to add .rpm or other binary files to git repos...)

File: docker/postlaunchconfig_update.sh (hunk 2 above)
Comment (AlvaroVega) [reply docker/postlaunchconfig_update.sh]: Fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/237/commits/0524907c39d60b11754f20f2674bc346cf9ee283

File: docker/postlaunchconfig.sh (hunk 1 above)
Comment (AlvaroVega) [reply docker/postlaunchconfig.sh]: fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/237/commits/2c8815cfce7c8997622ed1d6bf1215960200af85

File: docker/cracklib-devel-2.9.6-15.el8.x86_64.rpm
Comment (AlvaroVega) [reply docker/cracklib-devel-2.9.6-15.el8.x86_64.rpm:1]: This is right. Some unresolved deps are introduced in this way.

Comment (fgalan) [reply docker/cracklib-devel-2.9.6-15.el8.x86_64.rpm:1]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
Unsure about merge due to the fail in dockerhub...

**Pull request merged**